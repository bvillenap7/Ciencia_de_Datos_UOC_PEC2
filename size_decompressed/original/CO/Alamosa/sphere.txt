15 orbs in single file, equal distance apart heading south east.  ((Starlink satellites?));2020-05-03 05:00:00;8 minutes;15 orbs in single file, equal distance apart heading south east This morning (5/3/2020) about 5am, I spotted small, glowing, moving orbs in the sky . They were moving single file to the SE. A ll were equal distant apart except for orb #6, which lagged a bit behind.  The orbs behind #6 were all back to equal distance apart.  There were 15 orbs total, however there could have been more that I had not noticed previously to looking up.;37.4725;-105.8796
