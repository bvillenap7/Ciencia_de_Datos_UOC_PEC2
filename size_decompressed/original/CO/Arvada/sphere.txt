3 large blue spherical objects spotted in Arvada, Colorado in late 1986;1986-11-03 20:30:00;less than 1 minute;3 large blue spherical objects spotted in Arvada, Colorado in late 1986 The date and time are more of a guess at this point given that the event occured at least 2 decades ago.  Reading some of the recent events reminded me of it.   I was walking down the street and off to the west, toward the mountains I saw the large glowing blue spherical objects arc across the sky and then vanish.  They flew in-line with one another, like a traffic light, just straight up and down.  They just appeared, and followed, almost like a rainbow pattern and then vanished.   The lights remained solid the entire time, they did not flicker and they did not make any sound.  I would approximate the distance to have been at a minimum of 3 miles away, and still they were very large.  A few moments later, another individual came out of her house and said she saw them as well.  After about 20 minutes of continued observation of the location the objects were spotted, they did not reappear.;39.8289982352941;-105.110888235294
Watched 4 orbs of light traveling in unison then rotated around each other.  ((NUFORC Note:  We suspect advertising lights.  PD));2015-11-01 00:00:00;20 seconds;Watched 4 orbs of light traveling in unison then rotated around each other with one orb breaking free from pattern and rotating around the rest of the orbs.  I was driving, so I couldn't get video.  Tried to follow, but lost them. ((NUFORC Note:  We suspect advertising lights, although the witness believed that not to be the case.  PD));39.8289982352941;-105.110888235294
