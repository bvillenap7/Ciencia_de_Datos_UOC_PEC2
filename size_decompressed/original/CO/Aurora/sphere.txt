Spherical metallic object seen in broad daylight above University Hospital.;2007-10-15 16:00:00;22 minutes;Spherical metallic object seen in broad daylight above University Hospital. 16:00 – I’m a cancer researcher at the University of Colorado Health Sciences Center in Aurora, Colorado.   I left the research complex a bit early today and began walking to my car.  When I reached my car, I opened the door to get in, and I momentarily turned back toward the research buildings.  I saw a large object in the sky, and a quick comparison allowed me to estimate its altitude to be three times as high as the research building.  This was, of course, without knowing the ultimate size of the object, or the distance at which it appeared, so altitude was merely estimated.  My first thought was a helicopter, being on the campus of a university hospital, but there was no sound - absolutely no sound.  This caused me to look closer.  The object appeared to be quite large, larger than a helicopter, and spherical in shape.  Metallic silver best describes the color, with flashes of reddish-silver as the object caught the bright sunlight.  The object appeared to be hovering, slightly bobbing back, forth, up and down, as if it was a gigantic weightless balloon, tethered to the ground, accosted by the wind.  But the movements seemed different – as if they were defying gravity.  Up, then sideways, then quickly backward.  I’ve never seen any aircraft move that way.  Then, suddenly, the object shot upward and was nearly out of view within a few seconds.  I could still see the shining sunlight reflecting off of the object at its new altitude.  Suddenly, I was able to see a smaller, white object orbiting the object, which eventually shot off into the distance.  The object then ascended to an even higher altitude.  I sat down in my car, and rapidly opened my window.  The object was still visible.  I pulled out my phone and called my mother.  As she answered, I told her that I was seeing my first U.F.O. – and being a scientist, I wanted to rule out natural phenomena.  I mentioned the helicopter idea, but we both shot that down.  I mentioned an airplane – not likely.  A balloon?  It rose too quickly.  We couldn’t figure it out.  Then, as if on cue, a helicopter appeared from behind the hospital.  It passed through the general area where I had originally seen the unidentified object.  This was clearly a helicopter, it was smaller than the object, I could see the red and white paint, and I could make out the unmistakable sounds of the rotors.  It was a perfect controlled comparison.  Next, I was able to see a passenger airplane pass in the distance.  The shape, size, and sound were unmistakable.  Another perfect controlled comparison.  The object was still visible, a mere pinprick of bright light, moving slowly back and forth in the high atmosphere, reflecting sunlight every few seconds.  Suddenly, the object faded from view.  A few seconds later, I viewed a jet approaching the area where the object had been.  I could easily see the wings and shape of the aircraft, although I couldn’t make out the exact type.  The jet proceeded on its course then it disappeared on the horizon.  I waited a few more minutes, and the object appeared again.  Only for a few seconds, it quickly enlarged in size and then shrank off into the distance and disappeared for good.  It was 16:22 when I last saw the object.  I have no explanation for the events of this afternoon, although I can definitely say that the object was not a conventional aircraft. ((NUFORC Note:  Witness elects to remain anonymous  provides no contact information.  PD));39.6695444272445;-104.803991486068
Black orb flying incredibly low and fast at night in South East Aurora, CO, on July 22, 2014.;2014-07-22 23:59:00;1 second;Black Orb flying incredibly low and fast at night in South East Aurora, CO on July 22, 2014 On Tuesday, 7/22/2014, around midnight, my wife and I were driving south bound on Himalaya (in Aurora, Colorado) and just as we approached the intersection of Chenago, a dark shadowy object went east to west approximately 100' in front of us and 100' above. The object was flying incredibly fast, had no lights nor made any sound. The whole thing lasted only a second.  As it was going from left to right in my field of vision, about ¾ of the way across, the light from the street light (that was down and to the right) reflected off of it and it was like a camera flash. When it flashed, I got a tiny glimpse of the object, before it disappeared into the darkness. It appeared to be a round black ball and had a shine to the surface that caused the reflection.  I immediately asked my wife, “Did you just see that?”  She responded that she saw 'something' fly by and then the flash from the reflection off the street light. It was not a plane, bird, helicopter nor an alien UFO.  I believe it is the same object as seen on KDVR and CNN news reports from back in March of 2014 and November 2013 that was recorded twice from 84th & Federal.;39.6695444272445;-104.803991486068
Rainbow flashing Orb with Indigo Orbs flashing around the Main one.  ((anonymous report));2017-09-04 00:39:00;2 hours;Rainbow flashing Orb with Indigo Orbs flashing around the Main one. I was at my old house (I just moved three days ago)  Setting up My cameras and Tripod facing south in the sky.  I was preparing for my fifth skywatch that evening of September 4th, 2017, at 12:34 AM.  Since I was have been seeing this Blinking orb (What looked like to my naked eyes) I got my best camera with a night time setting to capture the object. I first noticed the object four nights ago.  It has been moving and hovering around the same section in the sky for the past few nights and I finally got the chance to see what it was. At first, I thought this object was a star, but it was moving too quickly and blinking/flashing rapidly.  Until I finally got a closer look. All it was doing was stopping at times, then moving to the right. It was not a straight line either. It would occasionally, as it moved to the right, move a little upward or downward.  It did this for two hours straight, I recorded it for 33 minutes with my cannon camera. I felt no fear.  If anything I was rather excited to see this.  I was filled with shock and awe looking at it move across the screen of the camera.  At times I could feel my body vibrating and burning from the inside.  A warm feeling throughout the whole body.  A bit of adrenaline, heart racing. I even actually lost sight of the object when I started to grow paranoid being outside by myself with no other people.  I did not know why, but after I felt this I packed my gear and headed inside to view the results.  I write Times and such in a record in my skywatch sessions.  I will paste my written session below. Day five of skywatching.   Times set during Skywatch - 12 am  through 1 am  Friday, September 4th, 2017,  Location - Colorado Springs  My backyard   12:34 AM Going out now.  I'm bored and cannot sleep.  I was outside earlier and spotted a big blue, moving orb above my house.  Might as well do a Skywatch tonight. 12:39 AM Just set up, spotted Blinky in the same sections of the sky the previous night. I'm naming the orb Blinky. 12:58 AM Filmed Blinky.  It moved behind the tree though.  It's in the usual spot, circling the sky miles away from my house.  Maybe just one mile.  The burning feeling is back.  Everything went silent.  I can hear the street cleaners.  Or this wubbing noise.  Freaked me out for a second there.  It's pretty loud. Feeling allot of adrenaline pumping through me.  I feel dizzy, hearts racing for some odd reason.  I'm not afraid though that's the thing.  I hear this machine hum in the sky above me. Someone said film and it was not me.  I feel calm suddenly after that voice. Blinky moved from behind the tree. 1:14 AM. Filming again, but I switched cameras. My video camera can pick up many flashing lights after I zoomed in in my video editor. All I do is crop it and I see the flashing lights. I cannot wait to see the picture/video results later. 3 minutes in of recording. We have some visible to my human eyes blinking flashing again.  Hmm... 1:41 AM Still stalking Blinky.  I got good footage of them. Still moving. I'm going to head inside now. 1:49 AM I'm inside now. I feel pretty joyful tonite. Let's take a gander at the results.  ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.6695444272445;-104.803991486068
