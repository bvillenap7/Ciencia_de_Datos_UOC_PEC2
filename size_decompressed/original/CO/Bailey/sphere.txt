Colorado 10/03/06;2006-10-03 19:00:00;10-15 minutes;Colorado 10/03/06 On the night of Tuesday 10/03/06 around sunset in Bailey, Colorado looking north, north/east. I noticed out the corner of my eye out the window these 2 lights hovering over the mountains. I went to look out the window figuring it was planes, and I was thinking, what are those planes doing so close to each other? Then I relized they werent moving at all. I went to get some binoculars for a better look and then I notced that each one had three diff. lights on the sides and infact were not moving. after watching them for about 10-15 minutes, they started to slowly follow eachother north and then quickly disseapeared., strange, I didnt know what to think of it!;39.4497;-105.4692
