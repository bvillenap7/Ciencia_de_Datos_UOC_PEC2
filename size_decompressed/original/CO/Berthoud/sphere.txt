Saw orb that faded from red to greenish blue, stayed perfectly still until attempt made to record video;2021-07-03 22:30:00;7 Minutes;Saw orb that faded from red to greenish blue, stayed perfectly still until attempt made to record video;40.2993;-105.1055
