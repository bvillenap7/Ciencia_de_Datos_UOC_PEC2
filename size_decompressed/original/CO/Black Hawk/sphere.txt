spheroid, leading edge shimmering red, main body whitish yellow, low altitude;1987-01-31 17:04:00;about 5 min;spheroid, leading edge shimmering red, main body whitish yellow, low altitude About the same time what had caused me to look up in the sky was a loud roar of a jet - clearly military by the sound of it.  The jet was going east to west and this sphere was low going west to east.  It didn't look like any weather balloon, no odd shapes dangling, or geometry.   It wasn't at all high up.  Hard to tell size, but I would say about 20 feet in diameter. It wasn't translucent, didn't change shape, or seem to bounce at all, but had a definite direction heading towards Lakewood, CO.   As it reached the front range, it looked like it turned back to the west, and I watched it slowly getting larger again until it was blocked by trees.   This is the second time I have seen something like this in the same approximate location and duration. - The previous event was in 2004.;39.8159;-105.4702
