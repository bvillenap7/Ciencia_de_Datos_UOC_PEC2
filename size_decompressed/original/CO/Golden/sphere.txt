Bright blue sphere emanating light appeared to move south and east across horizon.;1975-07-04 14:00:00;15 seconds;Bright blue sphere emanating light appeared to move south and east across horizon. Sometime in July of either 1974 or 1975, three observers were sitting at a stop sign on McIntrye at 32nd Avenue in Golden.  The car was facing south.  It was a clear day with no clouds and no rain coming from the west.   An object appeared in the horizon from the west and moved rapidly across the sky toward the east and south.  The object briefly hesitated about 3/4 way across the horizon and appeared to move more southward.   The object was a bright blue sphere that appeared almost transparent but also seemed to emanate light.  By size comparison, the object appeared to be the size of a full, harvest moon.  It is very difficult to determine the distance from the car to the object, but estimates are about 3 miles.   The observers were a male professional bank auditor a female college student working for an environmental consulting firm and a professional chemist and quality auditor.   The female student reported this incident at her workplace and discussed it with the firm's meteorologist.  It was the meteorologists opinion that the object was a plasma ball or ball lightening.   The female thought this not to be likely as the object appeared to be quite perfect and manufactured rather than a spontaneous natural occurrence.   My interest in reporting this sighting is to find out if there are any other individuals who witnessed this event. ((NUFORC Note:  Witness indicates that the date of the incident is approximate.  PD));39.7598265306123;-105.220002040816
Black ball in sky totally motionless no string;2011-11-29 14:30:00;10-15minutes;Black ball in sky totally motionless no string I was riding on the back of a motorcycle from Golden into Boulder, on CO state road 93, past a big, wide open field to my right.  Broad daylight, no clouds.  I saw what looked like a medicine ball, black, not shiny but matte, the way rubber is a dull black, perfectly round in the sky, not moving at all.  As the road curved slowly over the next few miles, I could see mostly all around it, and saw no string or pole to support it, nor any sort of propeller or engine that would indicate it was a toy, or any kind of protrusion that would show it to be a balloon or something like that.  I could see it for over ten minutes. ((NUFORC Note:  Witness elects to provide no contact information.  PD));39.7598265306123;-105.220002040816
